ietf/submit/checkers.py

```diff
@@ -280,6 +281,15 @@ def check_file_txt(self, path):
 
             dest = os.path.join(settings.SUBMIT_YANG_DRAFT_MODEL_DIR, model)
             shutil.move(path, dest)
+            ftp_dest = Path(settings.FTP_DIR) / "yang" / "draftmod" / model
```

Comment:
We write models when we do the checking? That seems odd. Shouldn't that happen when the draft is posted? (Not advocating a change as part of this PR, just wondering)